lib/tasks/scheduler.rake

```diff
@@ -0,0 +1,5 @@
+desc "Heroku scheduler file"
+
+task get_bridge_troll_class_count: :environment do
```

Comment:
The argument to `desc` should describe the following `task`, and there should be no newline between the methods.